### ISSUE TITLE
Feature request local RDP members

### ISSUE THREAD

**Issue by jackassplus** [BloodHoundAD/BloodHound#82]
While looping through to get sessions, check and see if there are any local RDP users who are not admins. 

Crap code for group members:
```

Function global:Get-LocalGroup{ 
#Requires -Version 2.0             
[CmdletBinding()]             
 Param              
   (                        
    [Parameter(Mandatory=$false, 
               Position=1,                           
               ValueFromPipeline=$true,             
               ValueFromPipelineByPropertyName=$true)]             
    [String[]]$ComputerName = "localhost", 
    [String]$LocalGroup = "Administrators"
   )#End Param 
Begin             
{             
      $MemberNames = @()      
}#Begin    
Process             
{ 
    $ComputerName | ForEach-Object { 
    $Computer = $_ 
        $Group = [ADSI]"WinNT://$Computer/$LocalGroup,group" 
        # if statement to check for errors if the local group does not exist. 
        if ($Group.Path) 
            { 
                $Members = @($Group.psbase.Invoke("Members")) 
                [Array]$MemberNames = $Members |  
                    ForEach-Object {$_.GetType().InvokeMember("Name", 'GetProperty', $null, $_, $null)} 
                    $Hash = @{ 
                        Server     = $Computer.ToUpper() 
                        LocalGroup = $LocalGroup 
                        Groups     = $MemberNames  
                    } 
                ForEach ($item in $Hash.Groups)
		{
		     $item
		}
            } 
    }#Foreach-Object(ComputerName) 
}#Process  
End 
{ 
}#End 
}#Get-LocalGroup

```
(stolen from https://gallery.technet.microsoft.com/scriptcenter/ed4a0cb7-603c-488d-afeb-194b7a60f42f#content and modified a bit)

**Comment by rvazarkar** [comment]
This is something we might add in the near future

**Comment by jeffmcjunkin** [comment]
Come to think of it, enumerating users with RDP access could be part of an attack path, if there's also a privesc available on the destination machine. 

Given Veil's PowerUp and the Empire integration, it may be possible often enough to warrant including RDP access in attack paths (BloodHound would have to assume privesc was possible) for tougher environments.

**Comment by rvazarkar** [comment]
This has been implemented in 2.0!
